Movies-MVC/Movies-MVC/Controllers/DetailViewController.swift

```diff
@@ -0,0 +1,262 @@
+        movieDetailScrollVew.addSubview(relizeTextLabel)
+        NSLayoutConstraint.activate([
+            relizeTextLabel.topAnchor.constraint(equalTo: relizeLabel.bottomAnchor),
+            relizeTextLabel.trailingAnchor.constraint(equalTo: relizeLabel.trailingAnchor),
+            relizeTextLabel.heightAnchor.constraint(equalToConstant: 30),
+            relizeTextLabel.widthAnchor.constraint(equalToConstant: 120)
+        ])
+    }
+
+    private func configureShowTizerButton() {
+        showTizerButton.setTitle(Constants.showTizer, for: .normal)
+        showTizerButton.backgroundColor = .systemBlue
+        showTizerButton.layer.cornerRadius = 10
+        showTizerButton.addTarget(self, action: #selector(showTizerAction), for: .touchUpInside)
+        showTizerButton.translatesAutoresizingMaskIntoConstraints = false
+        movieDetailScrollVew.addSubview(showTizerButton)
+        NSLayoutConstraint.activate([
+            showTizerButton.topAnchor.constraint(equalTo: relizeTextLabel.bottomAnchor, constant: 20),
+            showTizerButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            showTizerButton.widthAnchor.constraint(equalTo: posterImageView.widthAnchor),
+            showTizerButton.heightAnchor.constraint(equalToConstant: 40)
+        ])
+    }
+
+    private func configureDescriptionLabel() {
+        descriptionLabel.text = Constants.description
+        descriptionLabel.textAlignment = .center
+        descriptionLabel.font = UIFont.avenirNextDemiBold20()
+        descriptionLabel.translatesAutoresizingMaskIntoConstraints = false
+        descriptionLabel.setContentHuggingPriority(.defaultHigh, for: .vertical)
+        movieDetailScrollVew.addSubview(descriptionLabel)
+        NSLayoutConstraint.activate([
+            descriptionLabel.topAnchor.constraint(equalTo: showTizerButton.bottomAnchor, constant: 30),
+            descriptionLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            descriptionLabel.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.9),
+        ])
+    }
+
+    private func configureDescriptionTextLabel() {
+        descriptionTextLabel.text = movieDescription
+        descriptionTextLabel.textAlignment = .justified
+        descriptionTextLabel.numberOfLines = 0
+        descriptionTextLabel.translatesAutoresizingMaskIntoConstraints = false
+        descriptionTextLabel.setContentHuggingPriority(.defaultHigh, for: .vertical)
+        movieDetailScrollVew.addSubview(descriptionTextLabel)
+        NSLayoutConstraint.activate([
+            descriptionTextLabel.topAnchor.constraint(equalTo: descriptionLabel.bottomAnchor, constant: 30),
+            descriptionTextLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            descriptionTextLabel.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.9),
+        ])
+    }
+
+    private func configureCastLabel() {
+        if actorNames.count == 0 {
```

Comment:
```suggestion
        castLabel.text = actorNames.count == 0  ? Constants.noCast : Constants.cast
```
тут можно тернарный оператор использовать